src/test/ui/generic-associated-types/issue-90612.rs

```diff
@@ -0,0 +1,43 @@
+// check-pass
+
+#![feature(generic_associated_types)]
```

Comment:
I think both these tests would be better with the other normalize under binder tests. This one uses GATs, but is not a GATs issue. The other doesn't even use GATs.